src/ChemistryFeaturization.jl

```diff
@@ -107,8 +107,6 @@ end
 # update tests
 
 
-
-
```

Comment:
these two lines were causing a merge conflict while rebasing? that's odd, since I don't think I even touched upon this file in #56.